test-playwright.sh

```
@@ -0,0 +1,42 @@
+# Install dependencies.
+npm ci
+
+# Install Playwright browsers
+npm run playwright-install
+
+while getopts u:s: flag
+do
+  case "${flag}" in
+    u) ui=${OPTARG};;
+    s) spec=${OPTARG};;
+  esac
```

Shellcheck (0.11.0)

[warning] Invalid flags are not handled. Add a *) case.

(SC2220)